_sass/components/homepage/_infocols.scss

```diff
@@ -0,0 +1,70 @@
+.infocols-container {
+  max-width: 1280px;
```

Comment:
Can we use `$container-max-width` instead?

Reply:
done